fiorella/src/sections/footer.liquid

```diff
@@ -1,74 +1,85 @@
-  <ul>
-    {% for social in social_accounts %}
-      {%- assign social_handle = social | handleize -%}
-      {% capture social_link %}social_{{ social_handle }}_link{% endcapture %}
-      {% capture social_icon %}icon-{{ social_handle }}{% endcapture %}
-
-      {% if settings[social_link] != blank %}
-        <li>
-          <a href="{{ settings[social_link] }}" title="{{ 'layout.footer.social_platform' | t: name: shop.name, platform: social }}">
-            {% include social_icon %}
-            <span class="icon-fallback-text">{{ social }}</span>
-          </a>
-        </li>
-      {% endif %}
-
-    {% endfor %}
-  </ul>
-
   {% if section.settings.show_payment_icons %}
     {% unless shop.enabled_payment_types == empty %}
-      {%- assign payment_icons_available = 'amazon_payments,american_express,apple_pay,bitcoin,cirrus,dankort,diners_club,discover,dogecoin,dwolla,forbrugsforeningen,interac,jcb,litecoin,maestro,master,paypal,visa' | split: ',' -%}
-
-      <span class="visually-hidden">{{ 'layout.footer.payment_methods' | t }}</span>
+    {%- assign payment_icons_available = 'amazon_payments,american_express,apple_pay,bitcoin,cirrus,dankort,diners_club,discover,dogecoin,dwolla,forbrugsforeningen,interac,jcb,litecoin,maestro,master,paypal,visa' | split: ',' -%}
+    <span class="visually-hidden">{{ 'layout.footer.payment_methods' | t }}</span>
       <ul class="payment-icons">
         {% for type in shop.enabled_payment_types %}
           {% if payment_icons_available contains type %}
             <li>
-              {%- assign icon_name = type | prepend: 'icon-' -%}
-              {% include icon_name %}
-              <span class="visually-hidden">{{ type | capitalize | replace: '_', ' ' }}</span>
-            </li>
+                {%- assign icon_name = type | prepend: 'icon-' -%}
+                {% include icon_name %}
+                <span class="visually-hidden">{{ type | capitalize | replace: '_', ' ' }}</span>
+              </li>
           {% endif %}
         {% endfor %}
       </ul>
     {% endunless %}
   {% endif %}
 
-  {%- assign footer_linklist = section.settings.footer_linklist -%}
-  {% if linklists[footer_linklist].links.size > 0 %}
-    <ul>
-      {% for link in linklists[footer_linklist].links %}
-        <li><a href="{{ link.url }}">{{ link.title }}</a></li>
-      {% endfor %}
-    </ul>
-  {% endif %}
+  {%- assign social_accounts = 'Facebook_fiorella, Twitter, Pinterest_fiorella, Instagram, Snapchat, Tumblr, YouTube, Vimeo, Google_plus,' | split: ', ' -%}
 
-  <div>
-    <small>{{ 'layout.footer.copyright' | t }} &copy; {{ 'now' | date: "%Y" }}, {{ shop.name | link_to: '/' }}. {{ powered_by_link }}</small>
+  <div class="box-footer">
+    {%- for block in section.blocks -%}
+      <div class="box-footer-item hide-menu">
+        <h3 class="title-footer">{{linklists[block.settings.footer_linklist].title | upcase }}</h3>
```

Comment:
You're missing one space after the `{{`, no problem here, just wanted to point it out for you to be careful next time.